tools/sdk-cli/src/command_handlers.rs

```diff
@@ -232,29 +232,29 @@ pub(crate) async fn handle_command(
             let payment = sdk()?.payment_by_hash(hash).await?;
             serde_json::to_string_pretty(&payment).map_err(|e| e.into())
         }
-        Commands::Sweep {
+        Commands::RedeemOnchainFunds {
             to_address,
             sat_per_vbyte,
         } => {
             sdk()?
-                .sweep(SweepRequest {
+                .redeem_onchain_funds(RedeemOnchainFundsRequest {
                     to_address,
                     sat_per_vbyte,
                 })
                 .await?;
             Ok("Onchain funds were swept successfully".to_string())
```

Comment:
Maybe change this so we can see the txid
```suggestion
            serde_json::to_string_pretty(&resp).map_err(|e| e.into())
```

Reply:
Added